tools/cmake/app.cmake

```diff
@@ -19,3 +19,41 @@ endif()
 include(${MBED_PATH}/tools/cmake/profile.cmake)
 
 enable_language(C CXX ASM)
+
+# Find Python
+find_package(Python3 COMPONENTS Interpreter)
+include(${CMAKE_CURRENT_LIST_DIR}/CheckPythonPackage.cmake)
+
+# Check python packages from requirements.txt
+file(STRINGS ${CMAKE_CURRENT_LIST_DIR}/requirements.txt PYTHON_REQUIREMENTS)
+foreach(REQUIREMENT ${PYTHON_REQUIREMENTS})
+	
+	# Look for a string from the start of each line that does not contain "<", ">", "=", or " ".
+	if(REQUIREMENT MATCHES "^([^<>= ]+)")
+
```

Comment:
@multiplemonomials, Same comment as before, could you please remove the new line?